operations/payments.md

```diff
@@ -190,3 +190,279 @@ Adds a new alternative payment to a specified customer.
+| `ChargedUtc` | string | optional | Charged date and time of the payment in UTC timezone in ISO 8601 format. |
+| `CreatedUtc` | string | required | Creation date and time of the payment created in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the payment in UTC timezone in ISO 8601 format. |
+| `AccountingState` | string [Accounting item state](#accounting-item-state) | required | Accounting state of the payment. |
+| `State` | string [Payment state](#payment-state) | required | Payment state of the payment. |
+| `Identifier` | string | optional | Additional unique identifier of the payment. |
+| `PaymentType` | string [Payment type](#payment-type) | required | Payment state of the payment. |
+| `DataDiscriminator` | object [Payment item data](#payment-data-discriminator) | optional | Discriminator pointing to the field in the [Data](#payment-data) field that contains additional information. |
+| `Data` | object [Payment data](#payment-data) | optional | Additional payment data. |
+
+#### Payment data
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CardData` | object [Card data](#card-data)| optional | Contains additional data in the case of a card payment. |
+| `InvoiceData` | object [Invoice data](#invoice-data) | optional | Contains additional data in the case of an invoice payment. |
+
+#### Card data
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `PaymentCardId` | string | optional | Unique identifier of the payment card. |
+
+#### Invoice data
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `PaymentCardId` | string | optional | Unique identifier of the invoice. |
+
+#### Time interval
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+
+#### Accounting item state
+
+* `Open` - Accounting items which carry a non-zero value, are open, and have not been closed on a bill or invoice.
+* `Closed` - Accounting items which carry a non-zero value and have been closed on a bill or invoice.
+* `Inactive` - Accounting items which are either of zero value and have not been canceled, if the state of the payment item is Pending or Failed, or items of optional reservations. Until the reservation is confirmed, all its accounting items are Inactive.
+* `Canceled` - Accounting items which have been canceled, regardless of whether the item is of zero value.
+
+#### Payment state
+
+* `Charged`
+* `Canceled`
+* `Pending`
+* `Failed`
+* `Verifying`
+
+#### Payment type
+
+* `CreditCard`
+* `Invoice`
+* `Cash`
+* `Unspecified`
+* `BadDebts`
+* `WireTransfer`
+* `ExchangeRateDifference`
+* `ExchangeRoundingDifference`
+* `BankCharges`
+* `Cheque`
+* `Other`
```

Comment:
One of the main reasons why we needed the endpoint was to add the support for more payment types. as this was a strict enum in the accountingItems it can't be extended because it is a breaking change. Also some of the types can be added right away.
```suggestion
* `Other`
* ...
```